app/assets/javascripts/sufia/tabs.js

```diff
@@ -16,7 +17,11 @@ Blacklight.onLoad(function () {
     if (activeTab.length) {
       activeTab.tab('show');
     } else {
-      $('.nav-tabs a:first').tab('show');
+      var firstTab = $('.nav-tabs a:first');
+      // select the first tab if it has an id and is expected to be selected
+      if (firstTab[0].id != ""){
```

Comment:
@jcoyne The item exists on the Work listing, but the id is set to empty string. So it is defined, just not set to a valid value.